src/database/services/keycloak-api-service.ts

```diff
@@ -75,7 +78,110 @@ export const CreateKeycloakApiService = (): KeycloakApiService => {
         severaUserId: (user as CustomKeycloakProfile).severaUserId ?? undefined,
       } as CustomKeycloakProfile;
     },
-  };
+
+    /**
+     * Updates a user's attributes
+     * 
+     * @param id string
+     * @param attributes  Record<string, string[]>
+     */
+    updateUserAttribute: async (
+      id: string,
+      attributes: Record<string, string[]> 
+    ): Promise<void> => {
+      const bodyContent = {
+        attributes: attributes,
+      };
+
+      try {
```

Comment:
Looks good, however I now notice that the findUser method is missing the try catch handling, could you also set this up?